src/osemgrep/networking/Rule_fetching.ml

```diff
@@ -241,7 +241,8 @@ let import_callback ~registry_caching base str =
               *)
              let content =
                Lwt_platform.run
-                 (fetch_content_from_registry_url_async ~registry_caching url)
+                 (fetch_content_from_registry_url_async ~token_opt:None
```

Comment:
I don't think this should be `None`. `import_callback` likely needs an additional parameter, or `parse_rule` should take `import_callback` as a parameter so it doesn't need to take the token.

Reply:
Agreed. Would adding a `TODO` here with this implementation note suffice?

Reply:
Added a TODO onto the stack of todos above this method 😅